.github/workflows/quality.yml

```
@@ -73,6 +70,16 @@ jobs:
         REPO_NAME="${{ github.repository }}"
         echo "Code Coverage Report: https://codecov.io/gh/${REPO_NAME}/branch/${BRANCH_NAME}"
 
+    - name: Upload coverage report
+      uses: actions/upload-artifact@v3
```

actionlint (1.7.4)

the runner of "actions/upload-artifact@v3" action is too old to run on GitHub Actions. update the action's version to fix this issue

(action)